backend/oasst_backend/prompt_repository.py

```diff
@@ -229,6 +238,29 @@ def store_ranking(self, ranking: protocol_schema.MessageRanking) -> MessageReact
                     OasstErrorCode.TASK_PAYLOAD_TYPE_MISMATCH,
                 )
 
+    def insert_toxicity(self, message_id: UUID, model: str, toxicity: ToxicityClassification) -> MessageToxicity:
+        """Save the toxicity score of a new message in the database.
+        Args:
+            message_id (UUID): the identifier of the message we want to save its toxicity score
+            model (str): the model used for creating the toxicity score
+            toxicity (float): the values obtained from the message & model
+        Raises:
+            OasstError: if misses some of the before params
+        Returns:
+            MessageToxicity: the instance in the database of the score saved for that message
+        """
+
+        if None in (message_id, model, toxicity):
+            raise OasstError("Paramters missing to add toxicity", OasstErrorCode.GENERIC_ERROR)
+
+        message_toxicity = MessageToxicity(
+            message_id=message_id, model=model, score=toxicity.score, label=toxicity.label
```

Comment:
see now you added `label`, but it's missing in the `None` check above. I'm still quite convinced the `None` check is unnecessary and the pydantic validators will pull